generated/nirfsg/nirfsg/_library_interpreter.py

```diff
@@ -315,6 +315,20 @@ def configure_software_start_trigger(self):  # noqa: N802
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    def create_deembedding_sparameter_table_array(self, port, table_name, frequencies, sparameter_table, sparameter_table_size, number_of_ports, sparameter_orientation):  # noqa: N802
```

Comment:
Wow, that's a lot of parameteres.
I'm not asking for a change, just making an observation.